Data/Constants.cs

```diff
@@ -1,13 +1,45 @@
-﻿namespace DICUI.Data
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Media;
+
+namespace DICUI.Data
 {
     /// <summary>
-    /// Text for UI elements
+    /// Variables for UI elements
     /// </summary>
     public static class UIElements
     {
         public const string StartDumping = "Start Dumping";
         public const string StopDumping = "Stop Dumping";
         public const string FloppyDriveString = "<<FLOPPY>>";
+
+        private static IReadOnlyList<int> AllowedDriveSpeedsForCD { get; } = new List<int> { 1, 2, 3, 4, 6, 8, 12, 16, 20, 24, 32, 40, 44, 48, 52, 56, 72 };
+        private static IReadOnlyList<int> AllowedDriveSpeedsForDVD { get; } = AllowedDriveSpeedsForCD.Where(s => s <= 24).ToList();
+        private static IReadOnlyList<int> AllowedDriveSpeedsForBD { get; } = AllowedDriveSpeedsForCD.Where(s => s <= 16).ToList();
+        private static IReadOnlyList<int> AllowedDriveSpeedsForUnknownType { get; } = new List<int> { 1 };
+
+        public static IReadOnlyList<int> GetAllowedDriveSpeedsForMediaType(MediaType? type)
+        {
+            switch (type)
+            {
+                case MediaType.CD:
+                    return AllowedDriveSpeedsForCD;
+                case MediaType.DVD:
```

Comment:
DVD, HD-DVD, GC, Wii, and WiiU all (in theory) share these drive speeds

Reply:
Yes I didn't know it so I left them to be filled by you (or by me since you liste them all)